mkosi/config.py

```diff
@@ -3785,21 +3785,33 @@ def parse_config_one(self, path: Path, profiles: bool = False, local: bool = Fal
         extras = path.is_dir()
 
         if path.is_dir():
-            path = path / "mkosi.conf"
+            path /= "mkosi.conf"
```

Comment:
I generally like in place operators, but for paths I think their readability is not really great.

Reply:
Ehh we use them in other places already as well, let's keep em for now